src/components/QuizGeneratorModal.tsx

```diff
@@ -93,7 +93,7 @@ export default function QuizGeneratorModal(): JSX.Element {
     });
   };
 
-  const getCode = () => {
+  const getCode = useCallback(() => {
```

Comment:
I think this should be `useMemo`, not `useCallback`, and rather than calling `getCode` we should just say `const code = useMemo(...)`. Otherwise we'll end up calling the `getCode()` function every render anyway.

Reply:
Ah right, mb!